data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -44,15 +42,17 @@ private Utils() {
         // Hide constructor, as this is a static utility class
     }
 
-    static String post(String url, String payload, InputStream stream, long timeoutMs, Map<String, String> headers, boolean leaveOpen) throws DataServiceException, DataClientException {
+    static String post(String url, String payload, InputStream stream, long timeoutMs, Map<String, String> headers,
```

Comment:
It's weird that the httpclient is the value before the last.
I personally think it should be first.